app/models/post.rb

```diff
@@ -0,0 +1,12 @@
+class Post < ApplicationRecord
+  belongs_to :author, class_name: 'User', counter_cache: :posts_counter
+  has_many :comments
+  has_many :likes
+  after_save :update_posts_counter
+
+  private
+
```

Comment:
- [ ] It would be better if we address the comment from the previous code reviewer about creating the method

```
def update_posts_counter
  author.increment!(:posts_counter)
end
```
to update the post counter